xcm/xcm-builder/src/pay.rs

```diff
@@ -31,28 +31,20 @@ use xcm_executor::traits::{Convert, QueryResponseStatus, XcmQueryHandler};
 /// `PayOverXcm::pay` is asynchronous, and returns a `QueryId` which can then be used in
 /// `check_payment` to check the status of the XCM transaction.
 ///
-pub struct PayOverXcm<
-	DestChain,
-	Router,
-	Querier,
-	BlockNumber,
-	Timeout,
-	AccountId,
-	AccountIdConverter,
->(PhantomData<(DestChain, Router, Querier, BlockNumber, Timeout, AccountId, AccountIdConverter)>);
+pub struct PayOverXcm<Sender, Router, Querier, Timeout, Beneficiary, AssetKind>(
+	PhantomData<(Sender, Router, Querier, Timeout, Beneficiary, AssetKind)>,
+);
 impl<
-		DestChain: Get<xcm::v3::MultiLocation>,
+		Sender: Get<xcm::v3::MultiLocation>,
```

Comment:
can be `{parent: 0, X2(Parachain(1000), AccountId32(...))}`
for treasury case, AccounId32, is a treasury account on Statemint

Reply:
different now